### ISSUE TITLE
Error fugashi

### ISSUE THREAD

**Issue by ghost** [ogkalu2/comic-translate#27]
DLL load failed while importing fugashi: The specified module could not be found..

Last progress at forming TextBlocks. Image 1 of 4

I tried running this tool and it gave me this error, I have fugashi installed so i dont know why i got this

**Comment by ogkalu2** [comment]
Try running this in a virtual environment.

Open cmd in whatever directory you want the virtual environment installed (or cd path/to/virtual environment/folder)
then:
python -m venv comic-venv
comic-venv\Scripts\activate

Now go to the directory of the comic folder:

cd /path/to/comic-translate/

then
pip install -r requirements.txt
pip install fugashi
pip install pyperclip
pip install unidic_lite

python comic.py

Let me know how it goes.

**Comment by ayoosh007** [comment]
That didn't solve the problem.The wheels for fugashi are problematic so it is better to do a manual installation.I followed the steps based on an issue I found on their github repo:


"In general, this should be an issue of not having pre-built wheels for Python 3.10.
mecab.h and libmecab.lib cannot be found, causing building errors.

Eventually I noticed that I have to download [mecab-msvc-x64.zip](https://github.com/chezou/mecab/releases/download/mecab-0.996-msvc-5/mecab-msvc-x64.zip) [1](https://github.com/polm/fugashi/issues/44#user-content-fn-1-bfb7e5fb93d8178888472f33826fa092) and extract to the very specific location recorded in fugashi_utils.py[2](https://github.com/polm/fugashi/issues/44#user-content-fn-2-bfb7e5fb93d8178888472f33826fa092), in order to build the wheel and install successfully.
(ofc before this, have Build Tools and Windows SDK installed)

What I don't understand is that I have to use this specific (forked) version of mecab, instead of the installer from [Official Website](https://taku910.github.io/mecab/) (generally because installed ver have subfolders bin and sdk to store dll, h and lib separately). And the hardcoded path does look strange.

[fugashi-1.1.1-cp310-cp310-win_amd64.zip](https://github.com/polm/fugashi/files/7431601/fugashi-1.1.1-cp310-cp310-win_amd64.zip)

idk if this should be an issue, sorry for any incovenience"



I installed mecab first then used the fugashi wheel file to manually install it.That solved the problem

**Comment by KlausAD** [comment]
> That didn't solve the problem.The wheels for fugashi are problematic so it is better to do a manual installation.I followed the steps based on an issue I found on their github repo:
> 
> "In general, this should be an issue of not having pre-built wheels for Python 3.10. mecab.h and libmecab.lib cannot be found, causing building errors.
> 
> Eventually I noticed that I have to download [mecab-msvc-x64.zip](https://github.com/chezou/mecab/releases/download/mecab-0.996-msvc-5/mecab-msvc-x64.zip) [1](https://github.com/polm/fugashi/issues/44#user-content-fn-1-bfb7e5fb93d8178888472f33826fa092) and extract to the very specific location recorded in fugashi_utils.py[2](https://github.com/polm/fugashi/issues/44#user-content-fn-2-bfb7e5fb93d8178888472f33826fa092), in order to build the wheel and install successfully. (ofc before this, have Build Tools and Windows SDK installed)
> 
> What I don't understand is that I have to use this specific (forked) version of mecab, instead of the installer from [Official Website](https://taku910.github.io/mecab/) (generally because installed ver have subfolders bin and sdk to store dll, h and lib separately). And the hardcoded path does look strange.
> 
> [fugashi-1.1.1-cp310-cp310-win_amd64.zip](https://github.com/polm/fugashi/files/7431601/fugashi-1.1.1-cp310-cp310-win_amd64.zip)
> 
> idk if this should be an issue, sorry for any incovenience"
> 
> I installed mecab first then used the fugashi wheel file to manually install it.That solved the problem
> 
> Also note that the mecab file is a fork and thus must be manually installed.Don't use the official installer

Hey there, I am not very good with type of stuff so I have not been able to do the fix you mentioned, could you please dumb it down for me a little?

**Comment by ogkalu2** [comment]
Fugashi has not been a dependency for some time now so this wouldn't be a problem anymore. And there's installers now for Windows and MacOS, so no fiddling with python anymore.
